spring-cloud-aws-messaging/src/main/java/io/awspring/cloud/messaging/listener/SimpleMessageListenerContainer.java

```diff
@@ -428,6 +428,7 @@ public void run() {
 				}
 				catch (MessagingException messagingException) {
 					applyDeletionPolicyOnError(receiptHandle);
+					break;
```

Comment:
I'm wondering if this break should be executed only when `this.deletionPolicy == ON_SUCCESS` ?